### PR TITLE
tweaks audit policies

### PR DESCRIPTION
**Pull request by p0lyn0mial**
makes it more generic so that it can be used by the API servers.

### CODE REVIEW

File: pkg/operator/apiserver/audit/audit_policies.go [hunk 1]
```diff
@@ -33,7 +33,17 @@ func WithAuditPolicies(targetName string, targetNamespace string, assetDelegateF
...
 
-// getRawAuditPolicies returns a raw config map that holds the audit policies for the target namespaces
+// GetAuditPolicies returns a config map that holds the audit policies for the target namespaces and name
+func GetAuditPolicies(targetName, targetNamespace string) (*corev1.ConfigMap, error) {
+	rawAuditPolicies, err := getRawAuditPolicies(targetName, targetNamespace)
+	if err != nil {
+		return nil, err
+	}
+
+	return resourceread.ReadConfigMapV1OrDie(rawAuditPolicies), nil
```
Comment (tkashem) [inline pkg/operator/apiserver/audit/audit_policies.go:43]: do we have a `ReadConfigMapV1` version that returns an `error`?

Comment (p0lyn0mial) [reply pkg/operator/apiserver/audit/audit_policies.go:43]: I’m not aware of any, but this is a common practice. It is considered a programmer error when it fails

**Comment by p0lyn0mial** [comment]
/assign @sttts 

for approval

**Pull request merged**